src/concepts/choosing_a_dataset_mobile.md

```diff
@@ -2,19 +2,19 @@
 
 ## Products Overview
 
-Before you perform an analysis, it is important to identify the products that you want to include. You can select any of the following Mozilla's mobile products:
-
-| Product Name            | App Name            | OS      | Notes                       |
-| ----------------------- | ------------------- | ------- | --------------------------- |
-| Firefox Android         | `Fennec`            | Android |                             |
-| Firefox iOS             | `Fennec`            | iOS     |                             |
-| Focus Android           | `Focus`             | Android | Privacy browser             |
-| Focus iOS               | `Focus`             | iOS     | Privacy browser             |
-| Klar                    | `Klar`              | Android | German Focus release        |
-| Firefox for Fire TV     | `FirefoxForFireTV`  | Android |                             |
-| Firefox for Echo Show   | `FirefoxConnect`    | Android |                             |
-| Firefox Lite            | `Zerda`             | Android | Formerly Rocket (See below) |
-| Fenix (Firefox Preview) | `org_mozilla_fenix` | Android | Uses Glean (see below)      |
+Before you perform an analysis, it is important to identify the products that you want to include. You can select any of the following mobile products:
+
+| Product Name            | App Name              | OS      | Notes                                  |
+| ----------------------- | --------------------- | ------- | -------------------------------------- |
+| Firefox for Android     | `org_mozilla_firefox` | Android | formerly Fenix; uses Glean (see below) |
+| Firefox Android (old)   | `Fennec`              | Android | End-of-life; replaced by above         |
+| Firefox iOS             | `Fennec`              | iOS     |                                        |
+| Focus Android           | `Focus`               | Android | Privacy browser                        |
+| Focus iOS               | `Focus`               | iOS     | Privacy browser                        |
+| Klar                    | `Klar`                | Android | German Focus release                   |
+| Firefox for Fire TV     | `FirefoxForFireTV`    | Android |                                        |
+| Firefox for Echo Show   | `FirefoxConnect`      | Android |                                        |
+| Firefox Lite            | `Zerda`               | Android | Formerly Rocket (See below)            |
```

Comment:
Do any of the above use Glean?

Reply:
### Uses Glean:

Firefox for Fire TV
Firefox for Echo Show
Lockwise Android (going away?)
Lockwise iOS (also going away?)
Firefox Reality for Android

### Planning on integrating with Glean very soon:

Focus Android
Focus iOS
Klar? (discussion of whether or not to instrument Klar is in the works)

Reply:
For completeness, there is also Reference Browser for Android that sends Glean telemetry.

Fire TV, Echo Show, Firefox Reality, and Firefox for iOS all send both `core` pings and Glean telemetry, and KPI reporting still relies on the `core` pings.